src/Symfony/Component/Mime/WrappedMessage.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime;
+
+/**
+ * @author Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * @experimental in 4.3
+ */
+class WrappedMessage extends RawMessage
+{
+    private $originalMessage;
+
+    /**
+     * @param iterable|string $message
+     */
+    public function __construct(Message $originalMessage, $message)
```

Comment:
Why not a `RawMessage` instead of a `Message`?

Reply:
This was done for a historic reason, namely detection of support in `MessageConverter`. But this changed in the meantime. I think allowing `RawMessage` should be possible.

Reply:
Done.